src/projects/create-pull-request/create-pull-request-description.html

```diff
@@ -58,12 +60,17 @@
         --paper-input-container-color: var(--secondary-background-color);
         --paper-input-container-focus-color: var(--anchor-color);
       }
+      #metadata {
+        margin-top: 10px;
+      }
     </style>
 
     <paper-input class="title" label="Enter a pull request title" value="{{title}}"></paper-input>
     <markdown-input
       label="Enter a description for your pull request. Markdown is supported."
       markdown="{{description}}"></markdown-input>
+    <paper-checkbox id="metadata" checked="{{metadata}}">Add metadata to description</paper-checkbox>
+    <paper-tooltip for="metadata" offset="5">Metadata will include a direct link to the pull request on preview-code.com</paper-tooltip>
```

Comment:
I think the checkbox and description can be a little bit clearer. For example: `Add direct link on GitHub` and as description: `Include a link back to the pull request on preview-code.com inside the pull description on GitHub`